pkg/endpoint/policy.go

```diff
@@ -766,9 +767,12 @@ func (e *Endpoint) Regenerate(owner Owner, reason string) <-chan bool {
 				buildSuccess = false
 				scopedLog.WithError(err).Warn("Regeneration of endpoint program failed")
 				e.LogStatus(BPF, Failure, "Error regenerating endpoint: "+err.Error())
+				owner.SendNotification(monitor.AgentNotifyEndpointRegenerateFail,
```

Comment:
The return value of this function is being ignored - should we log at debug level if it fails? Potential downside of clogging the logs, though.

Reply:
I thought about it and initially decided against it. This is still low volume so I have nothing against logging as debug. Will add it.

Reply:
fixed